generators/app/templates/static/build/updateApiVersion.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+#Get Latest API version from sfdx instance
+INSTANCE_URL=$(sfdx force:org:display --json | jq -r '.result.instanceUrl')/services/data
+VERSIONS_LIST=$(curl $INSTANCE_URL)
+VERSION_LIST_LENGTH=$(echo $VERSIONS_LIST | jq '. | length' | jq '.-1') 
+LATEST_VERSION=$(echo $VERSIONS_LIST | jq -r ".[${VERSION_LIST_LENGTH}].version")
+
+# Changing delimeter and assigning variables for read function for XML
+read_package () {
+    local IFS=\>
+    read -d \< TAG VALUE
+}
+
+# Read xml and assign Manifest API value
+while read_package; do
```

Comment:
why even bother reading the current versions and comparing them with the latest?  

Seems much simpler to just blind write the newest version every time.

Reply:
less editing of the file was my thought.